### PR TITLE
Marine light medical belt

### PR DESCRIPTION
**Pull request by tanker4390**
## About The Pull Request
Light medical belt, available infinitely in clothing vendors. Can hold 15 slots of various medical items.

216548462-38bdd814-f702-40fb-81dd-ea7a593d8691
## Why It's Good For The Game
Gives marines the option to fumble around with more medical supplies. I thought the lack of a specialized medical belt among its pouch and other counterpart was odd.
## Changelog
:cl: Tanker
add: Added a lightweight medical belt for marines to use.

### CODE REVIEW

**Comment by Lumipharon** [comment]
I swear we've had this PR before. Isn't this almost exactly the same as a medical rig?

**Comment by ClosetedSkeletonGH** [comment]
dark medical belt rises

**Comment by Mrrpip** [comment]
[        /obj/item/reagent_containers/hypospray/autoinjector,](https://github.com/tgstation/TerraGov-Marine-Corps/blob/acd84e5c4dde86c7155cd23d00355dcd4e72db62/code/game/objects/items/storage/belt.dm#L100)

Balance concerns and redundancies aside. I believe this is already covered by 

https://github.com/tgstation/TerraGov-Marine-Corps/blob/acd84e5c4dde86c7155cd23d00355dcd4e72db62/code/game/objects/items/storage/belt.dm#L103

**Comment by Mrrpip** [comment]
As Github's weird with line pointing out, I believe it's the same with line 102 and 99. (Pill bottles and packets.)

**Comment by tanker4390** [comment]
> I swear we've had this PR before. Isn't this almost exactly the same as a medical rig?

Almost... It has less slots, and can't hold the same objects the med rig can.

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -82,6 +82,26 @@
...
+	can_hold = list(
+		/obj/item/healthanalyzer,
+		/obj/item/reagent_containers/glass/bottle,
+		/obj/item/reagent_containers/pill,
+		/obj/item/reagent_containers/syringe,
+		/obj/item/storage/syringe_case,
+		/obj/item/storage/pill_bottle,
+		/obj/item/stack/medical,
+		/obj/item/reagent_containers/hypospray,
+		/obj/item/reagent_containers/dropper,
```
Comment (QualityVan) [inline code/game/objects/items/storage/belt.dm:102]: I'm not wild about this in the first place, but if you're designing expanded medical storage for standard marines to use it needs to be limited to things standard marines should be using: packets, autoinjectors, stacks, and not a whole lot else. Total space is on the high end, but in keeping with pouches.

**Review by QualityVan — CHANGES_REQUESTED** [review]

**Comment by QualityVan** [comment]
Please don't mark things as resolved on your own, it makes rereviewing more of a nuisance.

Waiting on kuro for sprites.

**Comment by Lumipharon** [comment]
This is the equivilent of 10 mags worth of storage, this is super efficient storage for basic ungas. Not really a fan. If you want to go medic, go medic. This just lets a basic unga carry every med conceivable, although yes I know they can loot medical if they really want.

**Comment by yyzsong** [comment]
Honestly, I think ten slots would be a better fit. Fifteen slots for meds is almost as good as a medical storage rig, and is even pretty decent compared to the lifesaver bag (which has 21 slots)



> This is the equivilent of 10 mags worth of storage

What does this mean

**Comment by Losenis** [comment]
>Ammo belt is 2 ammo pouches + 1 slot, good
>Medical belt is 2 medical pouches + 1 slot, suddenly bad

I wish the average IQ being room temperature celsius only extended to within the game

**Review by Kuro020 — APPROVED** [review]

**Comment by QualityVan** [comment]
As mentioned, this is a lot of storage, but belt slot is generally important and this is on par with being about the same as both pouch slots. It's entirely possible that both this and the PFC-available pouches should have their size tuned down, but I'm leaving that for future considerations right now.

**Pull request merged**